packages/components/pie-checkbox/src/index.ts

```diff
@@ -125,11 +137,11 @@ export class PieCheckbox extends FormControlMixin(RtlMixin(LitElement)) implemen
             <input
                 type="checkbox"
                 .value=${value}
-                ?checked=${live(checked)}
+                .checked=${!!checked}
                 name=${ifDefined(name)}
                 ?disabled=${disabled}
                 ?required=${required}
-                .indeterminate=${indeterminate}
+                .indeterminate=${!!indeterminate}
```

Comment:
There was an IDE warning that `boolean | undefined` is not assignable to `boolean`.

Reply:
I think this can be addressed as part of the other discussion: https://github.com/justeattakeaway/pie/pull/1504/files#r1632938723